PoseEstimation-TFLiteSwift/NonMaximumnonSuppression.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  NonMaximumnonSuppression.swift
+//  PoseEstimation-TFLiteSwift
+//
+//  Created by Doyoung Gwak on 2020/04/15.
+//  Copyright © 2020 Doyoung Gwak. All rights reserved.
+//
+
+import Foundation
+
+class NonMaximumnonSuppression {
+    typealias MaximumPoint = (row: Int, col: Int, val: Float32)
+    
+    static func process(_ heatmap: TFLiteFlatArray<Float32>, partIndex: Int, width: Int, height: Int) -> [MaximumPoint] {
+        let filterSize = 3
+        var lastMaximumColumns: [MaximumPoint?] = (0..<width).map { _ in nil }
+        var results: [MaximumPoint] = []
+        
```

Comment:
I recommend to use collection's 'reserveCapacity'.

Reply:
@syjdev 
Thanks for your meaningful feedback.